sunbeam-python/sunbeam/commands/openstack.py

```diff
@@ -214,17 +214,16 @@ def run(self, status: Optional[Status] = None) -> Result:
         tfvars.update(
             {
                 "model": self.model,
-                # Make these channel options configurable by the user
-                "openstack-channel": OPENSTACK_CHANNEL,
-                "ovn-channel": OVN_CHANNEL,
-                "rabbitmq-channel": RABBITMQ_CHANNEL,
                 "cloud": self.cloud,
                 "credential": f"{self.cloud}{CREDENTIAL_SUFFIX}",
                 "config": {"workload-storage": MICROK8S_DEFAULT_STORAGECLASS},
                 "many-mysql": self.database == "multi",
             }
         )
         tfvars.update(self.get_storage_tfvars())
+        m = Manifest.load_latest_from_clusterdb(on_default=True)
+        LOG.debug(f"Latest manifest in openstack: {m}")
+        tfvars.update(m.get_tfvars(self.tfhelper.plan))
```

Comment:
I'm not sure about this merge order.

Do we want people to be able to override the name of the model ? it's hardcoded in many places.

Do we want them to be able to bootstrap a cluster with a single mysql server, only for them later applying a manifest with `"many-mysql": true` ?

I think we should introduce the idea of immutable keys, (you don't want to change the name of the model during the lifetime of the cluster). And we might to centralize all these configs to make it easier ? That should be in the defaults of the manifest I believe.

Reply:
As of now, the configs are not centralised. These configs wont be part of manifest.